### PR TITLE
Makefile, rdef, fix build, README

### PR DESCRIPTION
**Pull request by BachToTheFuture**
A necessary upstream patch for a recipe :)

### CODE REVIEW

File: Source/ZooKeeper.rdef [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	$"FFFFFFFFFFFBFBFBFB8A007DFFFFFFFF"
+	$"FFFFFFFFD100FBFBFB007D7DFFFFFFFF"
+	$"FFFFFFD1FF00FBFBB0007DF9F900FFFF"
+	$"FFFF00FFFF00FBFBB000F9F9F91000FF"
+	$"FFFFFF06FF00FBFBB0000000100000FF"
+	$"FFFFFFFFFFFF00FBB0000000000000FF"
+	$"FF00FFFFFFFFFFFF060606000006FFFF"
+};
+
+resource file_types message {
```
Comment (humdingerb) [inline Source/ZooKeeper.rdef]: Please move this on top of the icon resource.

File: Source/ZKApp.cpp [hunk 2]
```diff
@@ -13,6 +13,7 @@
 #include <Path.h>
 #include <Entry.h>
 #include <FindDirectory.h>
+#include <storage/FindDirectory.h>
```
Comment (humdingerb) [inline Source/ZKApp.cpp]: Are you 100% sure that's needed?
While you're at it, quickl alphabetically sort the includes.

File: Source/ZKApp.cpp [hunk 3]
```diff
@@ -389,7 +390,7 @@ ZooKeeperApp::MakeTempFolder(void)
 
 	status_t status;
 	BPath p;
-	if (find_directory(B_COMMON_TEMP_DIRECTORY, &p) != B_OK)	return B_ERROR;
+	if (find_directory(B_SYSTEM_TEMP_DIRECTORY, &p) != B_OK)	return B_ERROR;
```
Comment (humdingerb) [inline Source/ZKApp.cpp:392]: While you're updating constants, can you correct the path to Terminal in this file, line 368? It's "`/boot/system/apps/Terminal -t`"

File: Source/ZooKeeper.rdef [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+resource app_version {
+	major  = 2,
+	middle = 1,
+	minor  = 0,
+
+	variety = B_APPV_FINAL,
+	internal = 0,
+
+	short_info = "ZooKeeper",
```
Comment (humdingerb) [inline Source/ZooKeeper.rdef]: Change to "Tracker add-on to execute scripts", or something...

File: Source/ZooKeeper.rdef [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+	long_info = "ZooKeeper"
```
Comment (humdingerb) [inline Source/ZooKeeper.rdef]: Change to "ZooKeeper executes scripts on a set of files", or something...

**Review by humdingerb — CHANGES_REQUESTED** [review]
A few more things:
* Remove the file "DropZookeeperHere".
* Add "How to install" section in the ReadMe in line 14, saying to copy the addon to `~/config/non-packaged/add-ons/Tracker`.
* In the ReadMe "Example", change the path to `~/config/non-packaged/bin/psviewer`.
* In the ReadMe change the "BeOS" in the last line to "Haiku"

**Comment by scottmc** [comment]
Any chance you can wrap this one up today?  You've got a few pending pull requests...  Maybe we need to make sure they are closed before accepting your tasks as complete?

**Comment by BachToTheFuture** [comment]
@scottmc Oops, sorry! Forgot about this one :)

**Comment by BachToTheFuture** [comment]
@humdingerb @scottmc Done!

**Comment by BachToTheFuture** [comment]
@scottmc I'm really sorry for forgetting about this (and perhaps several other tasks) I'll keep in mind to recheck old pull requests as I do my new task... sometimes I get too absorbed in a new task and forget about them 😄
Thanks for the reminder.

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**